src/components/nodes/cards/stanford-event/stanford-event-card.tsx

```diff
@@ -30,13 +33,26 @@ const StanfordEventCard = ({node, headingLevel, ...props}: Props) => {
       aria-labelledby={node.id}
       isArticle
     >
-      <div aria-hidden className="flex flex-col items-start w-fit">
-        <div className="text-m0 font-semibold mb-4 w-full text-center">
-          {startMonth.toUpperCase()}
-        </div>
-        <div className="text-m4 font-bold w-full text-center">
-          {startDay}
-        </div>
+      <div className="flex w-fit justify-start flex-row items-center min-w-[9rem] h-90">
+        <time dateTime={dateTimeString} className="flex flex-col">
+          <span className="text-m0 font-semibold w-full text-center"> {startMonth.toUpperCase()}</span>
+          <span className="text-m4 font-bold w-full text-center">{startDay}</span>
+        </time>
+
+        {
+        (startDay != endDay)
+        ||
+        (startMonth != endMonth)
+        ? 
+        <>
+          <span className='relative font-normal leading-trim top-7 text-m0 px-03em' aria-hidden='true'>– to –</span><span className='sr-only'>to</span>
```

Comment:
Don't need `aria-hidden` here since the parent container already has aria-hidden